modules/algo/backtesting.py

```diff
@@ -25,12 +25,13 @@ class BackTesting:
     sellpoints = {}
     df = {}
 
-    def __init__(self, data: dict, config_module: ConfigModule, strategy: Strategy):
+    def __init__(self, data: dict, config_module: ConfigModule, strategy: Strategy, additional_data):
         self.config = config_module
         self.starting_capital = config_module.starting_capital
         self.currency_symbol = config_module.currency_symbol
         self.strategy = strategy
         self.data = data
+        self.additional_data = additional_data
```

Comment:
```suggestion
        self.additional_pairs_data = additional_data
```

Maybe changing the variable names to this (or something like it) makes it easier to understand what kind of data this is